packages/common/src/store/pages/chat/slice.ts

```diff
@@ -140,7 +143,20 @@ const slice = createSlice({
     ) => {
       // triggers saga
     },
-    createChatSucceeded: (state, action: PayloadAction<{ chat: UserChat }>) => {
+    createChatBlast: (
+      _state,
+      _action: PayloadAction<{
+        audience: ChatBlastAudience
+        contentId?: ID
+        contentType?: 'track' | 'album'
```

Comment:
nit: `'track' | 'album'` has gotta be a shared type somewhere right? 🤷

Reply:
you would think, but we actually do this a lot